.github/workflows/pythonwheels.yml

```diff
@@ -0,0 +1,110 @@
+name: Create Platform Wheels
+
+on: [push]
+
+jobs:
+  manylinux-x86_64-wheel:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+    steps:
+    - name: Checkout pandas
+      uses: actions/checkout@master
+      with:
+        repository: pandas-dev/pandas
+    - name: Checkout release repo for build script
+      uses: actions/checkout@master
+      with:
+        path: "pandas-release"
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Docker Build
+      run: |
+        docker run --rm -e PLAT=manylinux2010_x86_64 -e PYVER=${{ matrix.python-version }} -v `pwd`:/io quay.io/pypa/manylinux2010_x86_64 /io/pandas-release/manylinux_build.sh
+    - name: Install and verify wheel
+      run: |
+        python -m pip install numpy pytz python-dateutil
+        # cd to not install the local unbuilt pandas directory
+        mkdir temp
+        cd temp
+        python -m pip install pandas --no-index -f ../wheelhouse
+        python -c "import pandas"
+        cd ..
+    - name: Upload linux wheel artifact
+      uses: actions/upload-artifact@v1
+      with:
+        name: wheels
+        path: wheelhouse
+
+  macos-wheel:
+    runs-on: macos-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+    steps:
+    - name: Checkout pandas
+      uses: actions/checkout@master
+      with:
+        repository: pandas-dev/pandas
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install setuptools wheel cython numpy
```

Comment:
I don't think it's needed to install dependencies, the wheel building step should create an isolated environment with the dependencies listed in pyproject.toml?

Reply:
Maybe an issue elsewhere, but removing this fails to build:

https://github.com/WillAyd/pandas-release/pull/1/checks?check_run_id=444542968

Reply:
Ah, maybe I am mistaken and pyproject.toml is only used when using pip commands (like "pip install" when building from source). Sorry about that. 

In any case, if installing numpy, you need to install the oldest supported version to have compatible wheels for different numpy versions.